arangod/Statistics/ServerStatistics.h

```diff
@@ -24,18 +24,22 @@
 #ifndef ARANGOD_STATISTICS_SERVER_STATISTICS_H
 #define ARANGOD_STATISTICS_SERVER_STATISTICS_H 1
 
+#include <atomic>
+#include <memory>
+
 class ServerStatistics {
  public:
-  static ServerStatistics statistics();
+  friend std::unique_ptr<ServerStatistics> std::make_unique<ServerStatistics>(double&);
 
-  static void initialize();
+  static ServerStatistics& statistics();
+  static void initialize(double);
 
-  double _startTime;
-  double _uptime;
+  const double _startTime;
+  std::atomic<double> _uptime;
+  std::atomic<int> _intermediateCommits;
```

Comment:
This should not be an `int`. First, the value will never get negative, and second, the number of commits can get huge during the lifetime of the server process.